server/views/pages/search.html

```diff
@@ -1,60 +1,26 @@
-{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
-{% from "../components/feedback-widget/macro.njk" import hubFeedbackWidget %}
-{% extends "../components/template.njk" %}
-{% from "govuk/components/input/macro.njk" import govukInput %}
-{% from "govuk/components/button/macro.njk" import govukButton %}
-{% from "../components/user-details/macro.njk" import userDetails %}
+{% extends "../components/basicTemplate.njk" %}
 
-{% block pageTitle %}
-  {{ title }}
-{% endblock %}
-
-{% block head %}
-  <link href="/public/stylesheets/application.css" rel="stylesheet"/>
-{% endblock %}
-
-{% block header %}{% endblock %}
-{% block homeNavigation %}{% endblock %}
-{% block search %}{% endblock %}
-
-{% block main %}
-  <div class="govuk-width-container govuk-!-padding-top-5">
-    <div class="govuk-main-wrapper">
-      <div class="govuk-grid-row">
-        <div class="govuk-grid-column-full">
-          <div class="results">
-            {% if data[0] %}
-              <h3 class="govuk-heading-m">Showing results for "{{ query }}"</h3>
-              <ul class="govuk-list">
-                {% for item in data %}
-                  <li>
-                    <a class="govuk-link" href="{{item.url}}">
-                      <span class="govuk-!-font-weight-bold">{{ item.title }}</span>
-                      <p class="govuk-body">
-                        {% if item.summary %}{{ item.summary }}
-                        {% else %}No summary available{% endif%}
-                        </p>
-                      </a>
-                    </li>
-                  {% endfor %}
-                </ul>
-              {% else %}
-                <h3 class="govuk-heading-m">No results found for "{{ query }}"</h3>
-              {% endif %}
-            </div>
-          </div>
-        </div>
-      </div>
+{% block content %}
+  <div class="results">
+    {% if results[0] %}
+      <h3 class="govuk-heading-m">Showing results for "{{ query }}"</h3>
+      <ul class="govuk-list">
+        {% for item in results %}
+          <li>
+            <a class="govuk-link" href="{{item.url}}">
+              <span class="govuk-!-font-weight-bold">{{ item.title }}</span>
+              <p class="govuk-body">
+                {% if item.summary %}{{ item.summary }}
+                {% else %}No summary available{% endif%}
```

Comment:
Is it not cleaner to perform checks like this (`if item.summary....`) in the controller and just pass the result through to the view? Would make cases like this more easily testable and cleans up the view a little?